engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
It can be, but SecurityUtils conceived to hold utility functions is probably not a good place to put a setup method like this. Could be another class but that would be an overkill. I am fine either way.